packages/zero-react/src/use-query.tsx

```diff
@@ -30,10 +30,14 @@ export function useQuery<
   const view = viewStore.getView(
     z.clientID,
     q as AdvancedQuery<TSchema, TReturn>,
-    enable,
+    enable && z.server !== null,
```

Comment:
We already allow the server to be specified as null, so it seems like this should have already worked?